### DISCUSSION TITLE
Allow user registration via admin panel

### DISCUSSION THREAD

**Discussion by fahadshery** [dani-garcia/vaultwarden#2075] [Q&A]
Hi,

thank you for this great project!
I initially turned off the user registration. I wanted to create users via admin panel. However, if you turn off the user registration, you can't create users via admin section. I believe this should be allowed to manage your users and your own instance privately.
`This instance will be exposed to the internet and I don't want users to be able to login or create accounts.`
thanks.

**Comment by BlackDex** [comment]
That is not true. If you turn of user registration you can still create users via the admin interface.

**Comment by fahadshery** [comment]
> That is not true. If you turn of user registration you can still create users via the admin interface.

well this is weird. Pls see the image:
<img width="1437" alt="Screenshot 2021-10-30 at 14 41 45" src="https://user-images.githubusercontent.com/7301267/139535223-0ca7ddfe-51c2-448a-a279-4d0b69a3323a.png">
)
Here are the environment variables that I am setting:
```
ADMIN_TOKEN= my token
SIGNUPS_ALLOWED=false
INVITATIONS_ALLOWED=false
LOG_FILE=/data/access.log
DOMAIN=https://vault.example.com
```

**Comment by BlackDex** [comment]
Just enter an mail in the invite box and press invite

**Comment by BlackDex** [comment]
Though, you would need to enable invites though.

**Comment by fahadshery** [comment]
> Just enter an mail in the invite box and press invite

ok retested and it worked. It is also not allowing signups via the main signup page which is great! I think the admin panel should be a little more descriptive to let the admin know that they can create new users via invite only and they can not create new users directly within the database.

> Though, you would need to enable invites though.

even though the `INVITATIONS_ALLOWED=false`. It is still inviting the users. I mean I was able to send invite email and the user registered via creating an account after receiving the invite email.

**Comment by BlackDex** [comment]
Well, if we would have created that it shouldn't work, then we would have removed that from the interface.
